application/backend/src/main/resources/apps/typerefinery/components/forms/fields/select/clientlibs/behaviour.js

```diff
@@ -0,0 +1,11 @@
+window.Typerefinery = window.Typerefinery || {};
+window.Typerefinery.Components = Typerefinery.Components || {};
+window.Typerefinery.Components.Forms = Typerefinery.Components.Forms || {};
+window.Typerefinery.Components.Forms.Select = Typerefinery.Components.Forms.Select || {};
+
+;(function (ns, document) {
```

Comment:
As of now I don't think we need any clientlibs for the select component.

Reply:
Deleted from code.